pkg/cmd/clustertask/describe_test.go

```diff
@@ -0,0 +1,301 @@
+				tb.TaskRunStartTime(clock.Now().Add(-10*time.Minute)),
+				cb.TaskRunCompletionTime(clock.Now().Add(17*time.Minute)),
+				tb.StatusCondition(apis.Condition{
+					Status: corev1.ConditionTrue,
+					Reason: resources.ReasonSucceeded,
+				}),
+			),
+		),
+		tb.TaskRun("taskrun-2", "ns",
+			tb.TaskRunLabel("tekton.dev/task", "clustertask-one-everything"),
+			tb.TaskRunSpec(
+				tb.TaskRunTaskRef("clustertask-1", tb.TaskRefKind(v1alpha1.ClusterTaskKind)),
+				tb.TaskRunServiceAccountName("svc"),
+				tb.TaskRunInputs(tb.TaskRunInputsParam("myarg", "value")),
+				tb.TaskRunInputs(tb.TaskRunInputsResource("my-repo", tb.TaskResourceBindingRef("git"))),
+			),
+			tb.TaskRunStatus(
+				tb.TaskRunStartTime(clock.Now().Add(-10*time.Minute)),
+				cb.TaskRunCompletionTime(clock.Now().Add(17*time.Minute)),
+				tb.StatusCondition(apis.Condition{
+					Status: corev1.ConditionTrue,
+					Reason: resources.ReasonSucceeded,
+				}),
+			),
+		),
+		tb.TaskRun("taskrun-3", "ns",
+			tb.TaskRunLabel("tekton.dev/task", "clustertask-full"),
+			tb.TaskRunSpec(
+				tb.TaskRunTaskRef("clustertask-1", tb.TaskRefKind(v1alpha1.ClusterTaskKind)),
+				tb.TaskRunServiceAccountName("svc"),
+				tb.TaskRunInputs(tb.TaskRunInputsParam("myarg", "value")),
+				tb.TaskRunInputs(tb.TaskRunInputsParam("print", "booms", "booms", "booms")),
+				tb.TaskRunInputs(tb.TaskRunInputsResource("my-repo", tb.TaskResourceBindingRef("git"))),
+				tb.TaskRunOutputs(tb.TaskRunOutputsResource("my-image", tb.TaskResourceBindingRef("image"))),
+			),
+			tb.TaskRunStatus(
+				tb.TaskRunStartTime(clock.Now().Add(-10*time.Minute)),
+				tb.StatusCondition(apis.Condition{
+					Status: corev1.ConditionUnknown,
+					Reason: resources.ReasonRunning,
+				}),
+			),
+		),
+	}
+
+	ns := []*corev1.Namespace{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "ns",
+			},
+		},
+	}
+
+	cs, _ := test.SeedTestData(t, pipelinetest.Data{ClusterTasks: clustertasks, Namespaces: ns, TaskRuns: taskruns})
+	seeds = append(seeds, cs)
+
+	cs2, _ := test.SeedTestData(t, pipelinetest.Data{ClusterTasks: clustertasks, Namespaces: ns})
+	cs2.Pipeline.PrependReactor("list", "taskruns", func(action k8stest.Action) (bool, runtime.Object, error) {
+		return true, nil, errors.New("fake list taskrun error")
+	})
+
+	seeds = append(seeds, cs2)
+
+	testParams := []struct {
+		name        string
+		command     []string
+		input       pipelinetest.Clients
+		inputStream io.Reader
+		wantError   bool
+		want        string
+	}{
+		{
+			name:        "Describe with no arguments",
+			command:     []string{"describe", "notexist"},
+			input:       seeds[0],
+			inputStream: nil,
+			wantError:   true,
+			want:        "clustertasks.tekton.dev \"notexist\" not found",
+		},
+		{
+			name:        "Describe clustertask with name only",
+			command:     []string{"describe", "clustertask-justname"},
+			input:       seeds[0],
+			inputStream: nil,
+			wantError:   false,
+			want: `Name:   clustertask-justname
```

Comment:
Those are not needed anymore :angel:

Reply:
Yeah, just realized! I'll take them out.